chia/cmds/wallet_funcs.py

```diff
@@ -622,3 +622,18 @@ async def execute_with_wallet(
             print(f"Exception from 'wallet' {e}")
     wallet_client.close()
     await wallet_client.await_closed()
+
+
+async def create_did_wallet(args: Dict, wallet_client: WalletRpcClient, fingerprint: int) -> None:
+    name = args["name"]
+    amount = args["amount"]
+    if amount is None:
+        amount = 1
+    try:
+        response = await wallet_client.create_new_did_wallet(amount, name)
+        wallet_id = response["wallet_id"]
+        my_did = response["my_did"]
+        print(f"Successfully created a DID wallet with name {name} and id {wallet_id} on key {fingerprint}")
+        print(f"Successfully created a DID {my_did} in the newly created DID wallet")
```

Comment:
I could be mistaken, but I believe the `create_new_did_wallet()` call doesn't wait for the transaction to become confirmed on-chain. I think we'll probably want to handle this in a manner similar to `create_new_pool_wallet()`. Take a look at the `create()` command in `plotnft_funcs.py`: https://github.com/Chia-Network/chia-blockchain/blob/73e1fabb5dca522c27c16592bdc07c327c99abf6/chia/cmds/plotnft_funcs.py#L104

Reply:
Should we revisit that when we merge a recent main that has the up to date version of this code?